Documentation/Tools/HolographicRemoting.md

```diff
@@ -0,0 +1,59 @@
+# Getting Started with Holographic Remoting
+
+Holographic remoting streams holographic content from a PC to your Microsoft HoloLens in real-time, using a Wi-Fi connection. This feature can significantly increase developer productivity when developing mixed reality applications.
+
+## Initial setup
+
+To enable remoting to a HoloLens, it is important to ensure that the project is using the latest remoting components.
+
+1. Open **Window > Package Manager**
```

Comment:
I think it would be useful to add that you need to switch to UWP to use remoting since Standalone is the default:
```suggestion
1. Ensure the platform has been switched to **Univeral Windows Platform (UWP)** in the Build Settings
1. Open **Window > Package Manager**
```

Reply:
i made your recommended change in a different spot (in the connecting instructions)